src/jit/importer.cpp

```diff
@@ -7664,6 +7676,22 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
             }
             else
             {
+                if (fatPointerCandidate)
+                {
+                    // fatPointer candidates should be in statements of the formL call() or var = call().
```

Comment:
how about explaining why?

---
In reply to: [99698702](https://github.com/dotnet/coreclr/pull/9191#discussion_r99698702) [](ancestors = 99698702)